upgrade/steps-helpers.go

```diff
@@ -139,13 +139,6 @@ func say(msg string) func([]byte) (string, error) {
 	}
 }
 
-func getTfProviderRepoName(providerName string) string {
```

Comment:
Instead of relying on a manual mapping of pulumi provider names to the Tf provider name, or assuming they are equivalent, it should be set in the config file (or with `--upstream-provider-name`)